seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  res.status(200).json({ data: rows });
+});
+
+app.delete("/postsdelete", async (req, res) => {
+  const { postId } = req.body;
```

Comment:
```suggestion
app.delete("/posts/:postId", async (req, res) => {
  const { postId } = req.params;
```
1. 위 코멘트와 마찬가지로 엔드포인트에는 delete와 같은 동사(동작)가 들어가지 않도록 해주세요. http method 가지고 충분히 설명이 됩니다.
2. delete 메소드에는 body로 데이터를 보내지 않도록 해주세요. 이는 실제로 delete method를 사용할 때에는 body로 들어오는 데이터를 파싱하지 않도록 설계된 프레임워크나, 버전이 있기 때문에 꼭 사용해야한다면 유의하여 사용하되, 원칙적으로는 지양하는 것이 좋습니다. 의도와 다르게 동작할 수 있습니다.